Wizarr - DB migration fails

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Wizarr

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/wizarr.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📈 Which Proxmox version are you on?

pve-manager/8.4.13/5b08ebc2823dd9cb (running kernel: 6.8.12-14-pve)

### 📝 Provide a clear and concise description of the issue.

During the update process, the alembic DB migration fails, resulting in a broken installation.

### 🔄 Steps to reproduce the issue.

1. Take a snapshot
2. Run `update` in the LXC

### ❌ Paste the full error output (if available).

```
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 0271b864fd47 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 124ededf80dc is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 20250618b_drop_old_library_unique is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 0597748a6fef is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 14a87d76dff1 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 23bf17eccf13 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 490af4233e95 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 56b33a2ca88e is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 5805136a1d16 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 6c39692d6f32 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 7c76d64526a7 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 91865b5a23d2 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision 9e13976c0cfa is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision d0a4dfc55dc4 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision e3625bacf6f9 is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision e6006b5e374e is present more than once
  util.warn(
/opt/wizarr/.venv/lib/python3.13/site-packages/alembic/script/revision.py:213: UserWarning: Revision eb1f0c1fcdd4 is present more than once
  util.warn(
ERROR [flask_migrate] Error: Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads

[ERROR] in line 55: exit code 0: while executing command $STD /usr/local/bin/uv run flask db upgrade
```

### 🖼️ Additional context (optional).

_No response_

--- a/ct/wizarr.sh
+++ b/ct/wizarr.sh
@@ -39,20 +39,23 @@ function update_script() {
     msg_info "Creating Backup"
     BACKUP_FILE="/opt/wizarr_backup_$(date +%F).tar.gz"
     $STD tar -czf "$BACKUP_FILE" /opt/wizarr/{.env,start.sh} /opt/wizarr/database/ &>/dev/null
+    rm -rf /opt/wizarr/migrations/versions/*
     msg_ok "Backup Created"
 
     fetch_and_deploy_gh_release "wizarr" "wizarrrr/wizarr"
 
     msg_info "Updating $APP"
     cd /opt/wizarr
-    $STD /usr/local/bin/uv lock
-    $STD /usr/local/bin/uv sync --locked
-    $STD /usr/local/bin/uv run pybabel compile -d app/translations
+    $STD /usr/local/bin/uv sync --frozen
+    $STD /usr/local/bin/uv run --frozen pybabel compile -d app/translations
     $STD npm --prefix app/static install
     $STD npm --prefix app/static run build:css
     mkdir -p ./.cache
     $STD tar -xf "$BACKUP_FILE" --directory=/
-    $STD /usr/local/bin/uv run flask db upgrade
+    $STD /usr/local/bin/uv run --frozen flask db upgrade
+    if ! grep -q 'frozen' /opt/wizarr/start.sh; then
+      sed -i 's/run/& --frozen/' /opt/wizarr/start.sh
+    fi
     msg_ok "Updated $APP"
 
     msg_info "Starting $APP"

--- a/install/wizarr-install.sh
+++ b/install/wizarr-install.sh
@@ -23,13 +23,12 @@ fetch_and_deploy_gh_release "wizarr" "wizarrrr/wizarr"
 
 msg_info "Configure ${APPLICATION}"
 cd /opt/wizarr
-$STD /usr/local/bin/uv lock
-$STD /usr/local/bin/uv sync --locked
-$STD /usr/local/bin/uv run pybabel compile -d app/translations
+$STD /usr/local/bin/uv sync --frozen
+$STD /usr/local/bin/uv run --frozen pybabel compile -d app/translations
 $STD npm --prefix app/static install
 $STD npm --prefix app/static run build:css
 mkdir -p ./.cache
-$STD /usr/local/bin/uv run flask db upgrade
+$STD /usr/local/bin/uv run --frozen flask db upgrade
 msg_ok "Configure ${APPLICATION}"
 
 msg_info "Creating env, start script and service"
@@ -43,7 +42,7 @@ EOF
 cat <<EOF >/opt/wizarr/start.sh
 #!/usr/bin/env bash
 
-uv run gunicorn \
+uv run --frozen gunicorn \
     --config gunicorn.conf.py \
     --preload \
     --workers 4 \

